src/test/java/org/broadinstitute/hellbender/tools/walkers/annotator/MappingQualityRankSumTestUnitTest.java

```diff
@@ -71,7 +71,7 @@ public void testMQ(){
         Assert.assertEquals(ann.getKeyNames().get(0), key);
     }
 
-    @Test
+    @Test(enabled = false)  //To be consistent with annotate raw data now produces the the ranksum, making this test incorrect
```

Comment:
Should the test be adjusted to be correct? Do you have a plan/ticket for re-enabling this test?

Reply:
Whoops, I had thought I transferred all these. Well i'm moving the test to AS_MappingQualityRankSum and updating it.

Reply:
done